pandas/core/internals.py

```diff
@@ -1247,6 +1255,16 @@ def equals(self, other):
             return False
         return array_equivalent(self.values, other.values)
 
+    def quantile(self, values, qs, **kwargs):
+        if len(values) == 0:
+            return self._na_value
+
+        if com.is_list_like(qs):
+            values = [_quantile(values, x * 100, **kwargs) for x in qs]
```

Comment:
I see. Currently NOT because ``DataFrame.quantile`` use ``Series.quantile`` (not use added ``Block`` logic). Will take a look.